src/locals/de.ts

```diff
@@ -53,6 +73,13 @@ export default {
     login_successful: 'Sie haben sich erfolgreich angemeldet',
     login_failed: 'Username oder Passwort ist falsch.',
     login_incomplete: 'Bitte geben Sie Username und Passwort an.',
+    register_successful: 'Sie haben sich erfolgreich registriert',
+    register_failed: 'Sie konnten leider nicht registriert werden.',
+    verification_successful: 'Sie wurden erfolgreich verifiziert. Sie können sich nun anmelden und los tippen! :-)',
+    verification_failed: 'Leider konnte Ihr Account nicht verifiziert werden.',
+    forgot_password_failed: 'Leider kann ihr PW nicht zurückgesetzt werden.',
+    reset_password_successful: 'Password wurde erfolgreich zurückgesetzt',
+    reset_password_failed: 'Leider kann ihr PW nicht zurückgesetzt werden.',
   },
```

Comment:
What ist the difference between forgot_password_failed and reset_password_failed?